ci/tests/e2e-helm.sh

```diff
@@ -26,9 +26,11 @@ fi
 
 oc version
 
+echo $ROOTDIR
 make install
 
 deploy_operator() {
+    echo "ENTERED deploy_operator"
     kubectl create -f "$OPERATORDIR/deploy/service_account.yaml"
```

Comment:
it is no longer valid since these files are from the old layout. This func ought to be replaced by : 

````
if oc api-versions | grep openshift; then
        oc adm policy add-cluster-role-to-user cluster-admin -z nginx-operator || :
    fi
make install
make deploy IMG=$image
```